vnext/Folly/Folly.vcxproj.filters

```diff
@@ -183,10 +186,19 @@
     </ClInclude>
     <ClInclude Include="$(FollyDir)\folly\dynamic-inl.h">
```

Comment:
These files still exist but just include the new files to not break dependencies (see: https://github.com/facebook/folly/commit/cb479681ca490948d9b83a1ccff703e61c443fc4#diff-392b91b8b87bb9159b6158d7a291dad8884feabc65719bf737a010a6f7334511). Opted to keep them until these get removed.